test/controllers/locks_controller_test.rb

```diff
@@ -4,17 +4,24 @@
 SingleCov.covered!
 
 describe LocksController do
+  def create(resource = nil, options = {})
```

Comment:
a bit more verbose but maybe `create_lock_for` ? `create stage` doesn't appear to be related to locks